pom.xml

```diff
@@ -1353,7 +1353,9 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <includes>
+                        <!-- Tests classes should start with "Test", but we do also want to include tests incorrectly named, with "Test" at the end -->
                         <include>**/Test*.java</include>
+                        <include>**/*Test.java</include>
```

Comment:
Should we add a check which would fail a build if *Test.java is found instead?

Reply:
https://stackoverflow.com/questions/42341897/how-can-i-check-if-a-filename-in-my-maven-project-contains-certain-characters-a shows (not very elegant) way to achieve this.

Reply:
done